fullnode/prod/0.6.1/config/config.toml

```diff
@@ -0,0 +1,332 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+##### main base config options #####
+
+# TCP or UNIX socket address of the ABCI application,
+# or the name of an ABCI application compiled in with the Tendermint binary
+proxy_app = "tcp://127.0.0.1:28658"
+
+# A custom human readable name for this node
+moniker = "xxx"
+
+# If this node is many blocks behind the tip of the chain, FastSync
+# allows them to catchup quickly by downloading blocks in parallel
+# and verifying their commits
+fast_sync = true
+
+# As state sync is an experimental feature, this switch can totally disable it on core network nodes (validator, witness)
+state_sync_reactor = true
+
+# If this node is many days behind the tip of the chain, StateSyncHeight
+# allows them to catchup quickly by downloading app state (without historical blocks)
+# in parallel and start syncing block afterwards
+# <0 - turn off state sync
+# =0 - sync from peer's latest height
+# >0 - sync from that height
+state_sync_height = 0
+
+# Database backend: leveldb | memdb | cleveldb
+db_backend = "leveldb"
+
+# Database directory
+db_dir = "data"
+
+# Output level for logging, including package level options
+log_level = "statesync:info,blockchain:info,main:info,paramHub:info,dex:info,dexkeeper:info,state:info,token:info,pub:info,consensus:info,*:error"
+
+# Output format: 'plain' (colored text) or 'json'
+log_format = "plain"
+
+##### additional base config options #####
+
+# Path to the JSON file containing the initial validator set and other meta data
+genesis_file = "config/genesis.json"
+
+# Path to the JSON file containing the private key to use as a validator in the consensus protocol
+priv_validator_key_file = "config/priv_validator_key.json"
+
+# Path to the JSON file containing the last sign state of a validator
+priv_validator_state_file = "data/priv_validator_state.json"
+
+# TCP or UNIX socket address for Tendermint to listen on for
+# connections from an external PrivValidator process
+priv_validator_laddr = ""
+
+# Path to the JSON file containing the private key to use for node authentication in the p2p protocol
+node_key_file = "config/node_key.json"
+
+# Mechanism to connect to the ABCI application: socket | grpc
+abci = "socket"
+
+# TCP or UNIX socket address for the profiling server to listen on
+prof_laddr = ":9062"
```

Comment:
@huangsuyu hi, can prof_laddr be empty? I think there is no need for user to open this service.

Reply:
changed to nil